src/Dockerfile

```diff
@@ -3,8 +3,6 @@ ENV USER_UID=1001 USER_NAME=opslevel
 ENTRYPOINT ["/usr/local/bin/kubectl-opslevel"]
 WORKDIR /app
 RUN apt-get update && \
-    apt-get install -y curl && \
-    apt-get purge && apt-get clean && apt-get autoclean && \
-    curl -o /usr/local/bin/jq http://stedolan.github.io/jq/download/linux64/jq && \
-    chmod +x /usr/local/bin/jq
-COPY kubectl-opslevel /usr/local/bin
\ No newline at end of file
+    apt-get install -y curl jq && \
```

Comment:
Not sure if curl is still needed...

Reply:
@lboynton curl is probably not needed anymore